ee/fe/modules/300-prometheus/templates/prometheus/pp/prometheus.yaml

```diff
@@ -0,0 +1,256 @@
+{{- define "prometheus_resources" }} # for reference see modules/300-prometheus/hooks/detect_vpa_max.go
+cpu: 200m
+memory: 1000Mi
+{{- end }}
+
+{{- define "config_reloader_resources" }}
+cpu: 10m
+memory: 25Mi
+{{- end }}
+
+{{- if .Values.prometheus.internal.prometheusPlusPlus.enabled }}
+{{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+---
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: prometheus-pp
+  namespace: d8-monitoring
+  {{- include "helm_lib_module_labels" (list . (dict "app" "prometheus")) | nindent 2 }}
+spec:
+  targetRef:
+    apiVersion: "apps/v1"
+    kind: StatefulSet
+    name: prometheus-pp
+  updatePolicy:
+    updateMode: {{ .Values.prometheus.vpa.updateMode | quote }}
+  resourcePolicy:
+    containerPolicies:
+    - containerName: "prometheus"
+      minAllowed:
+        {{- include "prometheus_resources" . | nindent 8 }}
+      maxAllowed:
+        cpu: {{ .Values.prometheus.vpa.maxCPU | default .Values.prometheus.internal.vpa.maxCPU | quote }}
+        memory: {{ .Values.prometheus.vpa.maxMemory | default .Values.prometheus.internal.vpa.maxMemory | quote }}
+    - containerName: config-reloader
+      minAllowed:
+        {{- include "config_reloader_resources" . | nindent 8 }}
+      maxAllowed:
+        cpu: 20m
+        memory: 50Mi
+    {{- include "helm_lib_vpa_kube_rbac_proxy_resources" . | nindent 4 }}
+{{- end }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  name: pp
+  namespace: d8-monitoring
+  {{- include "helm_lib_module_labels" (list . (dict "app" "prometheus-pp" "prometheus" "pp")) | nindent 2 }}
+spec:
+  replicas: 1
+  retention: {{ .Values.prometheus.retentionDays }}d
+  retentionSize: {{ .Values.prometheus.internal.prometheusMain.retentionGigabytes }}GB
+  image: {{ include "helm_lib_module_image" (list $ "prometheuspp") }}
+  version: v2.53.2
+  enableRemoteWriteReceiver: true
+  enableFeatures:
+  - memory-snapshot-on-shutdown # https://ganeshvernekar.com/blog/prometheus-tsdb-snapshot-on-shutdown/
+  - extra-scrape-metrics # https://prometheus.io/docs/prometheus/2.44/feature_flags/#extra-scrape-metrics usage scrape_sample_limit
+  imagePullSecrets:
+  - name: deckhouse-registry
+  listenLocal: true
+  query:
+    maxSamples: 100000000
+    lookbackDelta: {{ mul (.Values.global.discovery.prometheusScrapeInterval | default 30) 2 }}s
+  additionalArgs:
+    - name: scrape.timestamp-tolerance
+      value: 10ms
+  initContainers:
+  - command:
+    - sh
+    - -c
+    - chown -vfR 64535:64535 /prometheus-db/
+    image: {{ include "helm_lib_module_common_image" (list . "alpine") }}
+    imagePullPolicy: IfNotPresent
+    name: fix-permissions
+    securityContext:
+      runAsNonRoot: false
+      runAsUser: 0
+    volumeMounts:
+    - mountPath: /prometheus-db
+      name: prometheus-pp-db
+      subPath: prometheus-db
```

Comment:
Is this required?
Seem to me you've used Prometheus resource from an old branch.